en/FeedEntity.md

```diff
@@ -0,0 +1,9 @@
+### FeedEntity
+
+| Field Name | Recommendation |
+| --- | --- |
+| id |  |
+| is_deleted | |
+| trip_update | When multiple trips in one block are included in one feed, it is recommended that their respective TripUpdate entities should be added to the feed entity in the same order that they are scheduled in the block (for more information about trips and blocks, please refer to [GTFS trips.txt](https://github.com/google/transit/blob/master/gtfs/spec/en/reference.md#tripstxt)). For example, trips with `trip_ids` 1, 2, and 3 that all belong to one block, and the vehicle travels trip 1, then trip 2, and then trip 3, the `trip_update` entities should appear in the feed in the order of trip 1, then trip 2, and then trip 3. This facilitates the storage of arrival and departure times by consumers that store information for an entire block of trips together. |

```

Comment:
This section is no longer up-to-date with changes made in [#206](https://github.com/google/transit/pull/206/).

Reply:
Removed in https://github.com/MobilityData/GTFS_Realtime_Best-Practices/pull/1/commits/9f4196f2d399d95eeae3f99eb8cf328efdc15dc8.